sites/standalone-swarm.conf

```diff
@@ -62,3 +60,38 @@ server {
         proxy_busy_buffers_size   256k;
     }
 }
+
+server {
+    include /etc/nginx/ssl.default.conf;
+
+    server_name
+        stats.zooniverse.org
+        stats-staging.zooniverse.org
+    ;
+
+    set $standalone_swarm_uri "https://standalone-swarm.zooniverse.org";
+    location / {
+        # Put the URL in a variable to force re-resolution of the DNS name
+        # Otherwise nginx will cache it indefinitely and it'll break if IPs change
+        resolver 172.17.0.2;
+        proxy_pass $standalone_swarm_uri;
+        proxy_set_header Host $host.web-hosts;
+        proxy_set_header X-Forwarded-Proto $http_x_forwarded_proto;
+        proxy_buffer_size   128k;
+        proxy_buffers   4 256k;
+        proxy_busy_buffers_size   256k;
+    }
+    
+    set $standalone_swarm_uri "https://standalone-swarm.zooniverse.org";
+    location /graphql {
```

Comment:
I'm not 100% sure how nginx processes these, so this might need to be above the other `location` section.